app/models/inventory_item.rb

```diff
@@ -1,2 +1,9 @@
 class InventoryItem < ActiveRecord::Base
+  using(:grossman)
+  after_initialize :init
```

Comment:
I see some "jiggery pokery" being involved which is why this is probably better left for when needed...